docs/best-practices.md

```diff
@@ -240,6 +240,26 @@ include::example$installation/post-installation-steps.sh[]
 ----
 ```
 
+### OCC Examples
+
+When creating examples that show how to use occ, ensure that you use the `occ-command-example-prefix` attribute.
+Doing so will keep all examples of its use consistent throughout the documentation.
+
+**Note:** when used within a source code block, as in the following example, `subs="attributes"` has to be set, otherwise it won't render properly:
+
+```asciidoc
+[source,console,subs="attributes"]
+....
```

Comment:
we have in all examples in the bpg `----`
this is the first place we use `....`
would be good to have that harmonized

Reply:
True. But I'll get that done in a different PR.

Reply:
really?
2x4 dots to be replaced by 2x4 dashes in a separate PR?

Reply:
Yes. Because if I'm replacing all of them, throughout the documentation, then that's a different PR. This PR is purely for standardising the occ command examples.

Reply:
https://github.com/owncloud/docs/pull/1239 has been created to standardise the use of source code block delimiters.